pkgs/stdenv/adapters.nix

```diff
@@ -93,8 +93,6 @@ rec {
   # Best effort static binaries. Will still be linked to libSystem,
   # but more portable than Nix store binaries.
   makeStaticDarwin = stdenv: stdenv.override (old: {
-    # extraBuildInputs are dropped in cross.nix, but darwin still needs them
```

Comment:
This is actually doing two things. First, this should be `buildPackages.targetPackages.darwin.CF`. Otherwise, static x86_64-darwin gets a dynamic CF instead of a static CF. Second, since that mirrors what `cross.nix` should have, I thought it would be cleaner to delegate to the override there instead of special casing for static.

Reply:
Are build- and targetPackages analogous to the build, host, target triumvirate? And if so, why wouldn't hostPackages suffice?

Reply:
There’s no `hostPackages`, but I can use `pkgsHostTarget` (assuming the current pattern isn’t somehow avoiding an infinite recursion).

Reply:
Wait, never mind. The way the cross stdenv is written doesn’t expose `pkgsHostTarget`. It only provides `buildPackages`.